finder.go

```diff
@@ -19,18 +19,25 @@ type Finder struct {
 type FinderOpts struct {
 	// Extensions controls which types of font files the finder reports.
 	Extensions []string
+	// SearchPaths is a list of paths to search for fonts.
+	SearchPaths []string
 }
 
 // NewFinder returns a new font finder. If the opts parameter is nil, default
 // options are used.
 //
 // Default options:
 //   Extensions: []string{".ttf", ".ttc", ".otf"}
+//   SearchPaths: xdg.FontDirs
```

Comment:
Might be good to add a link to the relevant section of the `xdg` documentation. Something like:
```
// Default options:
//   Extensions: []string{".ttf", ".ttc", ".otf"}
//   SearchPaths: xdg.FontDirs
//
// NOTE: See https://github.com/adrg/xdg#other-directories for more information
// about the default search paths.
```